### ISSUE TITLE
Set up Docker autobuild

### ISSUE THREAD

**Issue by JosephSalisbury** [giantswarm/mayu#23]

**Comment by teemow** [comment]
@JosephSalisbury I've changed the behaviour with the dockerfile here: 
https://github.com/giantswarm/mayu/pull/28

Now there is no Dockerfile in the root of the repository. Only if you `make bin-dist` you will have a Dockerfile in `bin-dist` and the tarball. 

That makes it easier to fetch the tarball, adapt the config and create your own image.

Does this match with your plans around automated builds/releases with `builder`?

**Comment by hectorj2f** [comment]
ping @JosephSalisbury

**Comment by JosephSalisbury** [comment]
Sorry, I completely missed this.

The make "interface" is that `make bin-dist` creates a directory named `bin-dist` with tarballs in (technically: a file suffixed with `.tar.gz`), and that those tarballs are then added to the GitHub Release.

Looking at the Makefile - and if I understand correctly, that the Mayu release is more of a "you'll need to take this and do some things" situation - and the `bin-dist` target, it looks like it all follows that interface.

tl;dr: looks good, fits with the plan for automated builds / releases. 

As a note, (and I'm not certain if this helps, just pointing it out), we have support for uploading multiple tarballs to a GitHub release starting in `builder v0.14.0` (see: https://github.com/giantswarm/builder#github-releases for more details). This might help with your 64/32-bit release issues <3

**Comment by hectorj2f** [comment]
So, let's close it then
